src/PKSim.Core/Model/PopulationSimulation.cs

```diff
@@ -279,5 +279,65 @@ public override void UpdateFromOriginalSimulation(Simulation originalSimulation)
          var sourcePopSimulation = originalSimulation as PopulationSimulation;
          sourcePopSimulation?.AdvancedParameters.Each(x => AddAdvancedParameter(x, generateRandomValues: true));
       }
+
+      private DataColumn aggregateDataColumns(IEnumerable<DataColumn> columns)
+      {
+         var column = columns.First();
```

Comment:
this will crash if there is no items. Clearly you are suspecting that this might be null see line below

Reply:
What does first mean? We need to comment this type of code BTW.
Do not comment the obvious, but if it is not obvious, comment